tasks/system_package_management.yml

```diff
@@ -0,0 +1,70 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+- name: Get ownCloud APT OpenPGP key
+  apt_key:
+    url: '{{ owncloud__apt_repo_key_url }}'
```

Comment:
The GPG key should be installed from an OpenPGP key server, not from the same source as the APT repository.